packages/now-cli/src/util/dev/server.ts

```diff
@@ -2216,3 +2243,45 @@ function hasNewRoutingProperties(nowConfig: NowConfig) {
     typeof nowConfig.trailingSlash !== undefined
   );
 }
+
+async function fetchAndDecryptEnvVars(
```

Comment:
Can we reuse the code from `vc env pull` instead of implementing again here?

This seems to be copy/pasted.